cli/command/image/save.go

```diff
@@ -54,3 +61,14 @@ func runSave(dockerCli command.Cli, opts saveOptions) error {
 
 	return command.CopyToFile(opts.output, responseBody)
 }
+
+func validateOutputPath(path string) error {
+	dir := filepath.Dir(path)
+	if dir != "" && dir != "." {
+		_, err := os.Stat(dir)
+		if err != nil && os.IsNotExist(err) {
+			return errors.New("directory does not exist")
```

Comment:
I think it should use `errors.Errorf()` (instead of `fmt`) otherwise sounds good to me.